### ISSUE TITLE
How could I set the rowHeight for the target single row?

### ISSUE THREAD

**Issue by xeoshow** [ANovokmet/svelte-gantt#222]
Hello, 

In our scenario, we will need to set the different rowHeight regarding different rows, is there any way for doing this?   

Just checked the api document, seems only get below, hopefully there could be some api for it.
```
rowHeight {Number} Height of a single row in px.
```

Thanks very much.

**Comment by xeoshow** [comment]
I found there seems got two places related as below, and both places got row.model context. 
Could we add row.model.height for the single row to support this feature? If it is defined, will use row.model.height, otherwise use rowHeight from options.

Thanks v much.

![image](https://github.com/ANovokmet/svelte-gantt/assets/1164284/35c2ce9d-ced4-4ae6-ab17-0c0f9f7f912c)

![image](https://github.com/ANovokmet/svelte-gantt/assets/1164284/60673fd1-14f2-40dd-8b1e-77cf94d233f8)

**Comment by ANovokmet** [comment]
Yes, sounds like a good feature.

**Comment by Mildclimate** [comment]
Hi @ANovokmet, 
I have implemented this, and tested various cases, seems works good.
I just modified as below:

File Path: 'src/modules/table/TableRow.svelte', line:24
https://github.com/ANovokmet/svelte-gantt/blob/005cec60fa43906c56693278420c035e62b11dc0/src/modules/table/TableRow.svelte#L24
```
style="height: {row.model.height || $rowHeight}px"
```

If it is OK, could you please merge it? Thanks.

**Comment by ANovokmet** [comment]
This needs more consideration: the current virtual scrolling implementation assumes all rows are of the same height. If rows are of different heights, it will result in rendering bugs (missing rows). See `rowContainerHeight` in `Gantt.svelte`.

**Comment by xeoshow** [comment]
Hi @ANovokmet, 

Thanks for reply, so is it still possible to be implemented? Besides the virtual scrolling, is there other aspects should be with more consideration? 

Best regards, Jason

**Comment by ANovokmet** [comment]
@xeoshow This is implemented in v4.4.0, which will be available soon.

**Comment by xeoshow** [comment]
Hi @ANovokmet , Thanks, and when will v4.4.0 be available on npmjs.com ?

Best regards, Jason
